<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QStackedWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="StartPage">
   <widget class="QFrame" name="frame_first_page">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
border-color: rgb(0, 0, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_first_page">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>501</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Tahoma&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Elevator Simulation</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
background-color: rgb(85, 170, 255);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="ParameterPage">
   <widget class="QComboBox" name="combobox_cat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 16pt &quot;Franklin Gothic Heavy&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Simple Elevator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modern Elevator(Multiple click)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azerbaijani Mode</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="simulate_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
background-color: rgb(85, 170, 255);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="info_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
image: &quot;C:\\Users\\Alex\\Downloads\\pngegg.png&quot;;
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	background-color: rgb(255, 0, 0);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Alex/Downloads/pngegg.png</normaloff>C:/Users/Alex/Downloads/pngegg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>427</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_floor">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Franklin Gothic Heavy&quot;;</string>
       </property>
       <property name="text">
        <string>How many floors?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lieedit_floor">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    font: 16pt &quot;Franklin Gothic Heavy&quot;;
	border : 1px solid black;
	border-radius :5px;
	paddling: 5px;
	color: white;
	
	background-color: rgb(170, 255, 255);
	

}

QLineEdit::focus{
	border: 2px solid blue;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="return_in_param">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	
	
	background-color: rgb(0, 170, 255);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Alex/Downloads/pngwing.com (1).png</normaloff>C:/Users/Alex/Downloads/pngwing.com (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="SimulationPage">
   <widget class="QDockWidget" name="pygame_dock">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>321</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QWidget" name="dockWidgetContents"/>
   </widget>
   <widget class="QFrame" name="elev_but_frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>431</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border: 2px solid green;
    border-radius: 4px;
    padding: 2px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="return_in_simul">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
font: 22pt &quot;Franklin Gothic Heavy&quot;;

border-radius: 25px;
	
	
	background-color: rgb(0, 170, 255);
color: rgb(85, 85, 255);

}

QPushButton:hover{
background-color: rgb(50, 110, 255);
color: rgb(255, 255, 255);

}

QPushButton:pressed{
background-color: rgb(100, 170, 255);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Alex/Downloads/pngwing.com (1).png</normaloff>C:/Users/Alex/Downloads/pngwing.com (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
